odoo/addons/base/data/res_country_data.xml

```diff
@@ -462,6 +462,7 @@
             <field file="base/static/img/country_flags/ec.png" name="image" type="base64" />
             <field name="currency_id" ref="USD" />
             <field eval="593" name="phone_code" />
+            <field name="vat_label">RUC</field>
```

Comment:
@jco-odoo here

Reply:
It is a minor improvement.  (but in the later versions it will use the identification types anyways (v15+))